test/unit/handlers/inbound-api.test.ts

```diff
@@ -69,20 +81,9 @@ jest.mock('@mojaloop/sdk-standard-components', () => {
     WSO2Auth: jest.fn(),
     Logger: {
```

Comment:
it would be nice to a shareable mock for this at some point.

Reply:
@lewisdaly : https://app.zenhub.com/workspaces/mojaloop-project-59edee71d1407922110cf083/issues/mojaloop/mojaloop/430